marker_uilints/tests/ui/item/print_me_root_module.stderr

```diff
@@ -0,0 +1,108 @@
+warning: printing item
+  |
+  = note: Mod(
+              ModItem {
+                  data: CommonItemData {
+                      id: ItemId(..),
+                      span: SpanId(..),
+                      vis: Visibility {{ /* WIP: See rust-marker/marker#26 */}},
+                      ident: Ident {
+                          name: "print_me_root_module",
+                          span: $DIR/print_me_root_module.rs:1:1 - 1:1,
+                      },
+                  },
+                  items: [
+                      Use(
+                          UseItem {
+                              data: CommonItemData {
+                                  id: ItemId(..),
+                                  span: SpanId(..),
+                                  vis: Visibility {{ /* WIP: See rust-marker/marker#26 */}},
+                                  ident: Ident {
+                                      name: "",
```

Comment:
This ident is suspiciously empty.shouldn't it be None?

Reply:
I'm not quite sure how this value was constructed. I've now removed all compiler generated items from Marker's AST representation and this "fixed" (meaning hid this issue) for now. I believe I have seen some circumstances, where the identifier was empty of something in the compiler for a valid reason, but I don't remember where. As for the `None` part. The `Ident` struct is used a lot, adding an option in it, would make handling it very cumber sum. So, even if this was a bug, I think it's better to have `""` as a symbol than using an option inside the `Ident`